rrdp_iostat.ml

```diff
@@ -424,16 +444,137 @@ module Iostats_value = struct
 			owner, ds_make ~name:(key_format "avgqu_sz")
 				~description:"Average I/O queue size"
 				~value:(Rrd.VT_Float value.avgqu_sz)
-                ~ty:Rrd.Gauge ~units:"requests" ~min:0. ();
+				~ty:Rrd.Gauge ~units:"requests" ~min:0. ();
 		]
 end
 
+module Blktap3_stats = struct
```

Comment:
Could this module be called something else, to avoid confusion with the Blktap3_stats module that contains the stubs? Blktap3_stats_wrapper maybe?